tests/unit/src/test/scala/tests/markdown/NestSuite.scala

```diff
@@ -231,7 +355,7 @@ class NestSuite extends BaseMarkdownSuite {
   )
 
   checkError(
-    "implicit-nok",
+    "implicit-nok".tag(SkipScala3),
```

Comment:
This is a new behaviour in Scala 3, I think, as implicits are resolved to nearest scope:

```scala
❯ cs launch scala3-repl
scala> object A {
     | implicit val x: Int = 1
     | object B {
     | implicit val y: Int = 1
     | object C{
     | implicitly[Int]
     | }
     |
     | }
     | }
// defined object A

scala> A.B.C
val res0: A.B.C.type = A$B$C$@8167f57
```

Reply:
This could be worth commenting next to the test.